libs/designsystem/src/lib/components/list/list.component.html

```diff
@@ -78,6 +180,8 @@
     [kirbyListItemColor]="getItemColor"
     [item]="item"
     [disabled]="isSwipingEnabled ? null : true"
+    [class.first]="isFirst(item)"
```

Comment:
This might still be more resilient than the first/last pseudo selectors and is just as performant (if not more)